src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, id, completed } = todo;
+  const [isEditing, setIsEditing] = useState(false);
+  const [changedTitle, setChangedTitle] = useState(title);
+
+  const handleChangeOfTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setChangedTitle(event.target.value);
+  };
+
+  const handleDelete = () => {};
+
+  const handleChangeSubmit = () => {};
+
+  const eventListenerKeyboard = () => {};
+
+  const switchCompleted = () => {};
+
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames('todo', { completed: completed })}
+    >
+      <label htmlFor={`checked-${id}`} className="todo__status-label">
+        <input
+          id={`checked-${id}`}
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={completed}
+          onChange={switchCompleted}
+        />
+      </label>
+
+      {!isEditing ? (
+        <>
+          <span
+            data-cy="TodoTitle"
+            className="todo__title"
+            onClick={() => setIsEditing(true)}
+          >
+            {title}
+          </span>
+
+          <button
+            type="button"
+            className="todo__remove"
+            data-cy="TodoDelete"
+            onClick={handleDelete}
+          >
+            ×
+          </button>
+        </>
+      ) : (
+        <form onSubmit={handleChangeSubmit}>
+          <input
+            data-cy="TodoTitleField"
+            type="text"
+            autoFocus
+            className="todo__title-field"
+            placeholder="Empty todo will be deleted"
+            value={changedTitle}
+            onChange={handleChangeOfTitle}
+            onBlur={() => {
+              handleChangeSubmit();
+              setIsEditing(false);
+            }}
+            onKeyUp={() =>
+              document.addEventListener('keyup', eventListenerKeyboard)
+            }
+          />
+        </form>
+      )}
+
+      {!todo && (
+        <div data-cy="TodoLoader" className="modal overlay">
+          <div className="modal-background has-background-white-ter" />
+          <div className="loader" />
+        </div>
+      )}
```

Comment:
try to remove this conditional rendering and always render it. The loader animation will appear when you add is-active class to this element.